modules/core/src/main/java/org/apache/ignite/spi/communication/tcp/TcpCommunicationSpi.java

```diff
@@ -2090,18 +2090,10 @@ private void dumpInfo(StringBuilder sb, UUID dstNodeId) {
         }
 
         if (connectionsPerNode > 1) {
-            connPlc = new ConnectionPolicy() {
-                @Override public int connectionIndex() {
-                    return (int)(U.safeAbs(Thread.currentThread().getId()) % connectionsPerNode);
-                }
-            };
+            connPlc = new RoundRobinConnectionPolicy();
```

Comment:
Why is it in this patch?

Reply:
Because they will look better in toString() with proper names. Instead of $1 and $2